packages/components/src/select-control/tags.js

```diff
@@ -68,17 +67,15 @@ class Tags extends Component {
 							/>
 						);
 					} ) }
+					{ searchInput }
 				</div>
-				{ showClearButton && (
+				{ showClearButton && selected.length > 1 && (
 					<Button
 						className="woocommerce-select-control__clear"
-						isLink
+						isLink={ false }
 						onClick={ this.removeAll }
 					>
-						<Icon
```

Comment:
I think this looks correct based on the original issue and the fact that tags also have an "x", but just to verify the intention is to remove the "x" icon for the clear button, right @LevinMedia?  cc @elizaan36